src/App.jsx

```diff
@@ -14,57 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selected, setSelected] = useState([]);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleClearSelection = () => setSelected('');
+  const handleRowClick = good => {
+    setSelected(selected === good ? '' : good);
```

Comment:
The `handleRowClick` function should toggle the selection correctly. Since `selected` is initially an array, you should check if `selected.includes(good)` and update the state accordingly, either by adding or removing the `good` from the array.